### PR TITLE
adding open assistant v2 dataset

### PR DESCRIPTION
**Pull request by vipulgupta1011**
Adding open assistant v2 dataset. Please let me know if any changes are needed

### CODE REVIEW

File: src/preparers.py [hunk 1]
```diff
@@ -199,6 +199,35 @@ def prepare_open_assistant(dset):
     return messages
 
 
+def prepare_open_assistant_v2(dset):
```
Comment (shayne-longpre) [inline src/preparers.py:202]: Is this different at all from the original? https://github.com/Data-Provenance-Initiative/Data-Provenance-Collection/blob/main/src/preparers.py#L173C8-L173C27

perhaps they could both call the same function?

**Review by shayne-longpre — APPROVED** [review]
Approved to merge after minor fix comment and also run tests and ensure there are no errors or warnings! Thank you!

File: src/preparers.py (hunk 1 above)
Comment (vipulgupta1011) [reply src/preparers.py:202]: Yeah, they both can call the same function

**Comment by vipulgupta1011** [comment]
Have made the change to call the same function.

**Pull request merged**